js_API/js/script.js

```diff
@@ -0,0 +1,40 @@
+document.querySelector("#input").addEventListener("keydown", (e) => {
+  if (e.key == "Enter")
+    apiRequest();
+});
+
+
+apiRequest = () => {
+
+  document.querySelector(".item").textContent = "";
+
+  // const value = 'melon';
+
+  const url = 'https://api.unsplash.com/search/photos?query=`+input.value+`&per_page=30&client_id=SouHY7Uul-OxoMl3LL3c0NkxUtjIrKwf3tsGk1JaiVo';
```

Comment:
```suggestion
  const url = 'https://api.unsplash.com/search/photos?query=`THIS`&per_page=30&client_id=SouHY7Uul-OxoMl3LL3c0NkxUtjIrKwf3tsGk1JaiVo';
```


input.value ---- это твоя строка из инпута

параметр , который мы передаем серверу, из которого он поймет, какие картинки тебенужны